.github/workflows/ci.yml

```diff
@@ -11,6 +11,27 @@ env:
   MYPY_FORCE_TERMINAL_WIDTH: "180"
 
 jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    timeout-minutes: 10
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
```

Comment:
any opinions on use `3.x` to use the latest stable (what does stable mean here?) python version? It's mentions in https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input